### PR TITLE
ui: fix dropdown styles

### PR DESCRIPTION
**Pull request by maxlang**
screen shot 2016-12-06 at 4 20 33 pm
screen shot 2016-12-06 at 4 20 14 pm

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/12118)

### CODE REVIEW

File: pkg/ui/styl/components/select.styl [hunk 1]
```diff
@@ -15,3 +17,13 @@
     display inline-block
     vertical-align middle
     width 200px
+
```
Comment (mrtracy) [inline pkg/ui/styl/components/select.styl]: These are not shameful, and they are in the "right place" as such; however, you should probably just move them to the shame file anyway.  That said, please put a comment here that "style overrides for react-select are in shame.styl due to excessive use of !important"

**Comment by tamird** [comment]
What is this screenshot? Perhaps you want to include a before/after?

**Comment by mrtracy** [comment]
LGTM, I had a good feeling this was possible with css only.

**Comment by mrtracy** [comment]
Instead of "before-after" screenshots, maybe just add some information to the commit message about what is being fixed.  Sorry, I didn't notice because I have a lot of context around this change, but its good practice to include more information in the commit message; a sentence would do.

**Comment by maxlang** [comment]
@tamird 

Before

screen shot 2016-12-06 at 4 33 51 pm
screen shot 2016-12-06 at 4 33 41 pm

Design

screen shot 2016-12-06 at 4 34 35 pm

**Comment by maxlang** [comment]
Actually I'll push the dropdown bit a bit lower to conform with the design.

**Comment by tamird** [comment]
+1 to meaningful commit messages.

**Comment by maxlang** [comment]
Changed commit message and fixed dropdown border/shadow/top.

screen shot 2016-12-07 at 2 45 02 pm
screen shot 2016-12-07 at 2 44 54 pm

**Comment by vivekmenezes** [comment]
LGTM

File: pkg/ui/styl/components/select.styl (hunk 1 above)
Comment (maxlang) [inline pkg/ui/styl/components/select.styl]: Should we just move all of them into `shame.styl`, or alternately a `select-shame.style` file so they're all together but still called out? I don't feel like putting all the select styles into the shame file is using shame correctly. My understanding is that it's more for "quick hacks" and things you intend to fix, but this feels a little more substantial and potentially long term (I don't like it but I don't intend to come back and fix it soon.)

**Comment by kuanluo** [comment]
We just talked about a few minor tweaks for the dropdown menu to make sure they're polished. @maxlang can you add the notes here to keep track?

**Pull request merged**